src/lib.rs

```diff
@@ -430,6 +550,26 @@ impl Navigator {
         }
     }
 
+    /// Reads angular velocity based on ICM20689 of [`Navigator`].
+    ///
+    /// Measurements in [degrees/sec]
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use navigator_rs::{Navigator};
+    /// use std::thread::sleep;
+    /// use std::time::Duration;
+    ///
+    /// let mut nav = Navigator::new();
+    /// nav.init();
+    ///
+    /// loop {
+    ///     let read = nav.read_gyro();
+    ///     println!("accel values: X={},Y={},Z={}", read.x, read.y, read.z);
```

Comment:
Put space after comma for better readability

Reply:
done.